packages/www/Dockerfile.dockerignore

```diff
@@ -0,0 +1,26 @@
+# Package to build
+packages/*
+!packages/api
+!packages/design-system
+!packages/www
+
+# General git and node frameworks files
+.git
+.github
+.cache
+**/node_modules
+**/coverage
+**/yarn-error.log
+**/.next
+
+# API-specific
+*/*/minio
+*/*/dist
+*/*/docs
+
+# WWW-specific
+*/*/.next
+*/*/out
+
+# So you can change this without forcing a rebuild
+Dockerfile
```

Comment:
nice 😆 

Shouldn't this be `**/Dockerfile` though?

Reply:
Good call.